app/allocation/index.js

```diff
@@ -33,7 +34,7 @@ router.get(
   protectRoute('allocation:create'),
   confirmation
 )
-
+router.get('/:allocationId/', protectRoute('allocations:view'), view)
```

Comment:
This doesn't need the trailing slash. Trailing slashes haven't been defined on any other routes in the application.